tests/cli_test.py

```diff
@@ -90,7 +97,7 @@ def test_simple(runner):
 
     result = runner.invoke(main_khal)
     assert not result.exception
-    assert result.output == 'No events\n'
+    assert strip_ansi_escapes(result.output) == 'No events\n'
```

Comment:
I'm not 100% sure. The doc doesn't say that bold escape codes are removed, but it *is* expected, so I'd still report this.

The problem is the inconsistency between python2 and python3, which *is* a [bug](mitsuhiko/click#334) IMHO.